synthtool/gcp/templates/node_library/.github/workflows/ci.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14, 16]
+        node: [{{metadata['engine'] | int}}, {{metadata['engine'] | int+2}}, {{metadata['engine'] | int+4}}, {{metadata['engine'] | int+6}}]
```

Comment:
Slight problem, node 18 won't exist for another 20 days.

Let's just make this `{{metadata['engine'] | int}}, {{metadata['engine'] | int+2}}, {{metadata['engine'] | int+4}}` for now, and in a couple months we can add Node 18.

Reply:
```suggestion
        node: [{{metadata['engine'] | int}}, {{metadata['engine'] | int+2}}, {{metadata['engine'] | int+4}}]
```